pkg/oci/promote.go

```
@@ -0,0 +1,127 @@
+	desc, err := store.Resolve(ctx, ref)
+	if err != nil {
+		return fmt.Errorf("resolving %s: %w", ref, err)
+	}
+
+	// 2. Fetch the manifest bytes.
+	rc, err := store.Fetch(ctx, desc)
+	if err != nil {
+		return fmt.Errorf("fetching manifest for %s: %w", ref, err)
+	}
+	manifestBytes, err := io.ReadAll(rc)
+	rc.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `rc.Close` is not checked

(errcheck)